### DISCUSSION TITLE
Question: How to specify a JDK if the project java files are using multiple JDK versions

### DISCUSSION THREAD

**Discussion by carina-lxx** [redhat-developer/vscode-java#3691] [Q&A]
I have the Java Language Server run on Java 17, and also trying to manually set `java.configuration.runtimes`
In the example you provided I see that it is possible to provide a list of JDK options. I have files expecting Java 8 also files expecting Java 11. How can I refer it to the correct JDK here?
`"java.configuration.runtimes": [
  {
    "name": "JavaSE-1.8",
    "path": "/path/to/jdk-8",
  },
  {
    "name": "JavaSE-11",
    "path": "/path/to/jdk-11",
  },
  {
    "name": "JavaSE-17",
    "path": "/path/to/jdk-17",
    "default": true
  },
]`

**Comment by snjeza** [comment]
You can install JDK 8, JDK 11, JDK 17 for your operating system and replace `/path/to/jdkXX` with your installed JDK's path.
